examples/dynamics/oscillator.ipynb

```diff
@@ -154,4 +156,4 @@
    "metadata": {}
   }
  ]
-}
```

Comment:
I agree, we should go with no linebreak for the time being. I think at a certain point, we should upgrade all out notebooks to the newer format (v4). I wouldn't be surprised if newer IPythons (or Jupyter) save with a newline at the end of the file, but if they don't, then we can vote / adopt a policy either way.